### PR TITLE
Fix security concept figure captions etc.

### PR DESCRIPTION
**Pull request by myidpt**
Small fixes following https://github.com/istio/istio.github.io/pull/1912.

### CODE REVIEW

**Comment by geeknoid** [comment]
You can ignore the bad link CI error, it's unrelated.

File: content/docs/concepts/security/index.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ In particular, Istio security mitigates both insider and external threats agains
 {{< image width="80%" ratio="56.25%"
     link="./overview.svg"
     alt="Istio security overview."
-    caption="Istio Security Architecture"
```
Comment (geeknoid) [inline content/docs/concepts/security/index.md]: If they're going to be the same, you don't need both alt and caption to be specified. You can just specify caption.

File: content/docs/concepts/security/index.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ Security in Istio involves multiple components:
 
 {{< image width="80%" ratio="56.25%"
     link="./architecture.svg"
-    alt="Istio architecture."
+    alt="Istio security architecture."
     caption="Istio Security Architecture"
```
Comment (geeknoid) [inline content/docs/concepts/security/index.md]: Same here. Please, no period on captions.

File: content/docs/concepts/security/index.md [hunk 3]
```diff
@@ -132,7 +132,7 @@ Currently we use different certificate key provisioning mechanisms for each scen
 
 ### on-premises machines scenario
 
-1. Citadel creates a gRPC service to take CSR requests.
+1. Citadel creates a gRPC service to take [CSRs](https://en.wikipedia.org/wiki/Certificate_signing_request) (Certificate Signing Requests).
```
Comment (geeknoid) [inline content/docs/concepts/security/index.md]: Should be the opposite ordering:

(Certificate Signing Requests)[...](CSRs)

File: content/docs/concepts/security/index.md (hunk 1 above)
Comment (myidpt) [reply content/docs/concepts/security/index.md]: I see. Thanks.

File: content/docs/concepts/security/index.md (hunk 2 above)
Comment (myidpt) [reply content/docs/concepts/security/index.md]: Sure.

File: content/docs/concepts/security/index.md (hunk 3 above)
Comment (myidpt) [reply content/docs/concepts/security/index.md]: Done.

**Pull request merged**